runner/operator/argos_operator/v1_2_0/argos_operator.py

```diff
@@ -267,7 +267,7 @@ def get_files_for_pairs(self, pairing):
     def get_files(self, request_id):
         files = FileRepository.filter(
             queryset=self.files,
-            metadata={settings.REQUEST_ID_METADATA_KEY: request_id, "igocomplete": True},
+            metadata={settings.REQUEST_ID_METADATA_KEY: request_id, "igoComplete": True},
```

Comment:
It is probably better to use igoComplete variable from settings.py https://github.com/mskcc/beagle/blob/master/beagle/settings.py#L430